taskcluster/ci/startup-test/kind.yml

```diff
@@ -30,6 +30,29 @@ task-defaults:
     run-on-tasks-for: []
 
 tasks:
+    fenix-arm-nightly-robo-opt:
+        attributes:
+            build-type: fenix-nightly
+            shipping-product: fenix
+        description: Fenix Nightly Opt Robo Test (ARM)
+        dependencies:
+            signing: signing-apk-fenix-nightly
+        run:
+            secrets:
+                - name: project/mobile/firefox-android/fenix/firebase
```

Comment:
Is it fine to use this firebase secret with a non-firebase task?

Reply:
These new tasks still require the same setup as others in that they require the same authentication to run on Firebase Test Lab, so yes.